cockatrice/src/dialogs/dlg_settings.cpp

```diff
@@ -524,7 +524,7 @@ UserInterfaceSettingsPage::UserInterfaceSettingsPage()
     setLayout(mainLayout);
 }
 
-void UserInterfaceSettingsPage::setNotificationEnabled(int i)
+void UserInterfaceSettingsPage::setNotificationEnabled(QT_STATE_CHANGED_T i)
 {
     specNotificationsEnabledCheckBox.setEnabled(i != 0);
```

Comment:
If this isn't an int, how can we compare against 0?

Reply:
i guess because c++ enums are just ints under the hood

Reply:
yes, they get cast automatically, most things are ints under the hood